docs/tutorials/quickstart.md

```diff
@@ -71,21 +71,12 @@ PUBLIC_URL_PREFIX= # Set Tunnel URL if using local storage
 7. Start the `kafka` and `postgres` container.
 ```bash
 $ bash scripts/run.sh kafka postgres
```

Comment:
We don't need to start kafka and postgres separately. We can start everything together.